rust/ql/lib/codeql/rust/frameworks/stdlib/core.model.yml

```diff
@@ -32,9 +31,9 @@ extensions:
       # Pin
       - ["core::pin::Pin", "Argument[0]", "ReturnValue", "value", "manual"]
       - ["<core::pin::Pin>::new", "Argument[0]", "ReturnValue", "value", "manual"]
-      - ["<core::pin::Pin>::new_unchecked", "Argument[0].Reference", "ReturnValue", "value", "manual"]
+      - ["<core::pin::Pin>::new_unchecked", "Argument[0]", "ReturnValue.Field[core::pin::Pin::pointer]", "value", "manual"]
```

Comment:
:+1:

I think the model above (for `<core::pin::Pin>::new`) should be updated similarly.

And the one above that (for `core::pin::Pin`) is probably mistaken.